### DISCUSSION TITLE
모듈 문제

### DISCUSSION THREAD

**Discussion by suuu111** [storycraft/node-kakao#409] [Q&A]
안녕하세요 node-kakao 사용하다가 모듈중에 갑자기 이런 기능이 changeProfile , kickUser , removeKicked 등 먹히지 않아 이렇게 질문하게 되었습니다. 혹시 해결방법 있을까요? 혹시 visual Studio Code가 문제일것 같아서 channel.hideChat()을 써봤지만 아래 오류가 뜨면서 강제 종료가 됩니다..

`UnhandledPromiseRejectionWarning: Error: Session closed before response #35`

**Comment by generalxd** [comment]
저도 이 증상입니다.

저녁쯤 채팅많아지는 시간대에 유독 자주보입니다.

**Reply by suuu111** [reply]
끊기는것두 문제인데 보통 Vsc에서 몇글자만 입력하면 자동으로 보이는데 그것도 보이지 않더라구요

**Reply by generalxd** [reply]
그거야 타입을 지정해주면되는문제입니다.

CLIENT.on('chat', TalkChatData, channel : TalkOpenChannel | TalkChannel)
